src/main/java/se/sundsvall/incident/service/CategoryService.java

```diff
@@ -5,64 +5,92 @@
 import static se.sundsvall.incident.service.mapper.Mapper.toCategoryDto;
 import static se.sundsvall.incident.service.mapper.Mapper.toCategoryEntity;
 
+import java.util.ArrayList;
 import java.util.List;
 
 import org.springframework.stereotype.Service;
 import org.zalando.problem.Problem;
 
-import se.sundsvall.incident.api.model.CategoryDTO;
+import se.sundsvall.incident.api.model.Category;
 import se.sundsvall.incident.api.model.CategoryPatch;
 import se.sundsvall.incident.api.model.CategoryPost;
-import se.sundsvall.incident.integration.db.CategoryRepository;
+import se.sundsvall.incident.api.model.ValidCategoryResponse;
+import se.sundsvall.incident.api.model.ValidOepCategoryResponse;
 import se.sundsvall.incident.integration.db.entity.CategoryEntity;
+import se.sundsvall.incident.integration.db.repository.CategoryRepository;
 import se.sundsvall.incident.service.mapper.Mapper;
 
 @Service
 public class CategoryService {
-
-	private static final String CATEGORY_NOT_FOUND = "Category with id: %s not found";
+	private static final String ENTITY_NOT_FOUND = "%s with id: %s not found";
+	private static final String CATEGORY = "Category";
```

Comment:
CATEGORY is only used in the ENTITY_NOT_FOUND string. Include it in the string right away?